### ISSUE TITLE
50 Cent Bulletproof (PaL-Unk) [SLES-53734] Green lines on Hardware mode

### ISSUE THREAD

**Issue by shandawgy** [PCSX2/pcsx2#259]
When you start game play in hardware mode there is green lines going in a vertical direction all over the screen. It seems this is a major problem and is hard to fix as it has happened in previous revisions and looks to be a plugin problem.

pcsx2-v1.2.1-511-g02b19b7

Plugins:
GSDx 20140818023333(MSVC 18.00,AVX) 0.1.16 [GSdx32-AVX]
XPad 0.2.0
SPU2-X 2014081802333 2.0.0
cdvdGigaherz (20140818023333ll) 0.9.0

My specs are:
Intel core i5-3470T CPU @ 2.90GHz
2x NVidia 560 GForce graphics card SLI
8gb Ram
Windows 8 64bit

**Comment by gregory38** [comment]
Could you try the build on this thread http://forums.pcsx2.net/Thread-GSdx-2-0
Note OpenGL only

**Comment by prafullpcsx2** [comment]
Confirmed fixed with OpenGL.

**Comment by prafullpcsx2** [comment]
@gregory38 Here is a gs dump for you from the cutscene.

http://www.filedropper.com/50cents

**Comment by gregory38** [comment]
```
as well as DX.
```
Ref partially port my texture shuffle code to DX. Code is enabled if CRC level is low enough. Anyway I need to debug it.

**Comment by gregory38** [comment]
For the history:
```
Opengl shows a bug as cutscenes are not being displayed if Depth is enabled.
```
I merged the PR #718

**Comment by refractionpcsx2** [comment]
My DX port is a major hack, wrong and basically i think it's just bypassing most of the code.  Need somebody who knows what they're doing (like @gabest11 *cough*) to do it properly
